glide.lock

```diff
@@ -135,10 +138,11 @@ imports:
 - name: github.com/robfig/cron
   version: 9585fd555638e77bba25f25db5c44b41f264aeb7
 - name: github.com/Sirupsen/logrus
```

Comment:
We should remove this old logrus entry with the uppercase "Sirupsen".

Reply:
It seems at least docker and go-ansiterm are using the uppercase version of "Sirupsen".
Removing it manually the uppercase entry is re-added after running `glide update`